src/Session/Handler/FileHandler.php

```diff
@@ -13,22 +15,33 @@
  */
 class FileHandler implements \SessionHandlerInterface
 {
+    /** @var integer */
+    protected $depth;
+    /** @var integer */
+    protected $mode;
     /** @var string */
     protected $savePath;
     /** @var \Symfony\Component\Filesystem\Filesystem */
     protected $fs;
+    /** @var \Psr\Log\LoggerInterface */
+    protected $logger;
 
     /**
      * Constructor.
      *
-     * @param string     $savePath Path of directory to save session files.
-     *                             Default null will leave setting as defined by system temp directory.
-     * @param Filesystem $filesystem
+     * @param string          $savePath   Path of directory to save session files.
+     *                                    Default null will leave setting as defined by system temp directory.
+     * @param Filesystem      $filesystem
+     * @param LoggerInterface $logger
      */
-    public function __construct($savePath = null, Filesystem $filesystem = null)
+    public function __construct($savePath = null, Filesystem $filesystem = null, LoggerInterface $logger = null)
```

Comment:
Can we flip logger and filesystem? We are way more likely to want to pass in a logger vs a filesystem